app/views/CancelConfirmView.scala.html

```diff
@@ -0,0 +1,74 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    p: components.p,
+    h1: components.h1
+)
+
+@(form: Form[_])(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("cancelConfirm.title"))) {
+
+    @formHelper(action = routes.CancelConfirmController.onSubmit(request.ern, request.arc), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
```

Comment:
You'll need to call @heading() here, as currently the heading is below the paragraphs.